samples/multicore/hello_world/src/main.c

```diff
@@ -6,16 +6,13 @@
 
 #include <zephyr/kernel.h>
 #include <zephyr/sys/printk.h>
-#include <zephyr/logging/log.h>
-LOG_MODULE_REGISTER(app);
 
 int main(void)
 {
-	int cnt = 0;
+	unsigned int cnt = 0U;
 
 	while (1) {
-		LOG_INF("test %d", cnt++);
-		printk("Hello world from %s\n", CONFIG_BOARD);
+		printk("Hello world from %s (%u)\n", CONFIG_BOARD, cnt++);
```

Comment:
Some tests might depend on the current format. I suggest restraining from changes beyond necessary.

Reply:
dropped